sciLifeLab_utils/run_delivery.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import glob
+import re
+import subprocess
+
+def main(arg):
+    project = os.path.split(os.path.realpath(arg.source))[1]
+    move_from_path = "{}/*/results/".format(arg.source)
+    pathway = glob.glob(move_from_path)
+    pattern = re.compile(r"/[^/]+/[^/]+/[^/]+/[^/]+/[^/]+/([^/]+)/") #Matches the group betwen the 6th / /
```

Comment:
@Hammarn : It is probably cause of the trailing `'/'` in the end. This would work

```
os.path.basename( os.path.dirname(path.rstrip('/')))
```